Tigran Kirakosov/flexbox_version/css/style.comp.css

```diff
@@ -0,0 +1,160 @@
+/*  MEDIA QUERY MANAGER

```

Comment:
You shouldn't add results css to git - adding scss files is enough

Reply:
Reasonable, compiled results are not informative anyway.